### PR TITLE
Feature/faqs

### PR DESCRIPTION
**Pull request by jeevanjoseph**
Added an FAQ section

### CODE REVIEW

File: deploy/basic/FAQs.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+    [Learn more](https://docs.cloud.oracle.com/iaas/Content/FreeTier/resourceref.htm) about Oracle Cloud Infrastructure's Always Free resources.
+
+2. **I get an error stating** `Permissions granted to the object storage service in this region are insufficient to execute this policy` **when I try to apply the terraform configuration.**
+
+    This is an edge case we are addressing and in the meantime, simply retrying the the `terraform apply` should resolve it.
+
+If you see something issue thats not listed here or have a question for us, please [open an issue](https://github.com/oracle/oci-quickstart-cloudnative/issues/new) and we will get back to you.
```
Comment (mvandervliet) [inline deploy/basic/FAQs.md]: Maybe add one more with:

**How do I delete the sample application?**

File: deploy/basic/FAQs.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+    [Learn more](https://docs.cloud.oracle.com/iaas/Content/FreeTier/resourceref.htm) about Oracle Cloud Infrastructure's Always Free resources.
+
+2. **I get an error stating** `Permissions granted to the object storage service in this region are insufficient to execute this policy` **when I try to apply the terraform configuration.**
+
+    This is an edge case we are addressing and in the meantime, simply retrying the the `terraform apply` should resolve it.
```
Comment (junior) [inline deploy/basic/FAQs.md]: Typo ..the the...
The instructions are for Resource Manager. 'terraform apply' appear that you are giving instructions for the terraform cli. Should have the menu breadcrumb ...stack > Terraform Actions > Apply

File: deploy/basic/FAQs.md (hunk 1 above)
Comment (junior) [inline deploy/basic/FAQs.md]: that's

File: deploy/basic/FAQs.md (hunk 2 above)
Comment (jeevanjoseph) [reply deploy/basic/FAQs.md]: done

File: deploy/basic/FAQs.md (hunk 1 above)
Comment (jeevanjoseph) [reply deploy/basic/FAQs.md]: done

Comment (jeevanjoseph) [reply deploy/basic/FAQs.md]: Should be fixed now

**Pull request merged**